app/app/models/users/student.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import Column, ForeignKey, Integer
+from sqlalchemy.orm import relationship
+
+from app.db.base_class import Base
+from app.models.term import Term
+from app.models.users.user import User
+
+
+class Student(Base):
+    user_id = Column(
+        Integer,
+        ForeignKey(f'{User.__table__.name}.id', ondelete='CASCADE'),
+        index=True,
+        primary_key=True,
+        nullable=False,
```

Comment:
doesn't `primary_key` add an implicit `nullable=false`?

Reply:
Yeah it should